### PR TITLE
Run specific services list for AWS

### PR DESCRIPTION
**Pull request by ganeshjagdale9545**
This PR regarding run specific services for AWS
in request we need to add new key in request body
'services': 'kms, rds, iam'
like this
i have tested this logic successfully

### CODE REVIEW

File: cartography/intel/aws/util/common.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 def parse_and_validate_aws_requested_syncs(aws_requested_syncs: str) -> List[str]:
     validated_resources: List[str] = []
-    for resource in aws_requested_syncs.split(','):
+    for resource in aws_requested_syncs:
```
Comment (mpurusottamc) [inline cartography/intel/aws/util/common.py:8]: same as gcp, services would be a comma separated string like "iam,ec2,s3". Let's use the split function to split here.

**Review by mpurusottamc — CHANGES_REQUESTED** [review]
minor code changes.

File: cartography/cli.py [hunk 2]
```diff
@@ -531,6 +531,7 @@ def run_aws(request):
         neo4j_max_connection_lifetime=request['neo4j']['connection_lifetime'],
         credentials=request['credentials'],
         params=request['params'],
+        aws_requested_syncs=request.get('services', None),
```
Comment (mpurusottamc) [inline cartography/cli.py:534]: same as gcp, we should use the default services list, if `services` parameter is null.

**Review by mpurusottamc — COMMENTED** [review]
one more change needed

File: cartography/cli.py (hunk 2 above)
Comment (ganeshjagdale9545) [reply cartography/cli.py:534]: for this we already have logic in `__init.py__` file

**Review by mpurusottamc — APPROVED** [review]
looks good.

**Pull request merged**